recipes/aws-parallelcluster/meta.yaml

```
@@ -0,0 +1,67 @@
+{% set name = "aws-parallelcluster" %}
+{% set version = "2.3.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "d2759cc8724586ce75b566e1ef7d499740d4f8cf5d417cf52f24e47f5b6ca180"
+
+build:
+  number: 0
+  entry_points:
+    - pcluster = pcluster.cli:main
+    - awsbqueues = awsbatch.awsbqueues:main
+    - awsbhosts = awsbatch.awsbhosts:main
+    - awsbstat = awsbatch.awsbstat:main
+    - awsbkill = awsbatch.awsbkill:main
+    - awsbsub = awsbatch.awsbsub:main
+    - awsbout = awsbatch.awsbout:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - boto3 >=1.9.54
+    - future >=0.16.0,<=0.17.1
+    - pip
+    - python
+    - tabulate >=0.8.2,<=0.8.3
+    - setuptools
+    - configparser
+  run:
+    - boto3 >=1.9.54
+    - future >=0.16.0,<=0.17.1
+    - python
+    - tabulate >=0.8.2,<=0.8.3
+    - setuptools
+    - configparser
+
+test:
+  imports:
+    - awsbatch
+    - pcluster
+    - tests
+    - tests.pcluster
+  commands:
+    - pcluster --help
+    - awsbqueues --help
+    - awsbhosts --help
+    - awsbstat --help
+    - awsbkill --help
+    - awsbsub --help
+    - awsbout --help
+
+about:
+  home: "https://github.com/aws/aws-parallelcluster"
+  license: "Apache Software"
```

We prefer the SPDX identifiers:
```suggestion
  license: Apache-2.0
```
(Check that it is 2.0)